src/subscription_manager/i18n.py

```diff
@@ -36,7 +36,7 @@
 LOCALE.language = None
 LOCALE.lang = None
 
-BLACKLISTED_LOCALES = ['turkish', 'tr_TR', 'tr_CY', 'ku_TR']
+BLOCKLISTED_LOCALES = ['turkish', 'tr_TR', 'tr_CY', 'ku_TR']
```

Comment:
See https://bugzilla.redhat.com/show_bug.cgi?id=1857214, in particular my comment 2.